src/util/evalUnits.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
```

Comment:
Aside comment: This should be shared with all JS developers at parity if there is not something available already.